lib/guard/rspec/command.rb

```diff
@@ -19,13 +19,26 @@ def initialize(paths, options = {})
       private
 
       def _parts
-        parts = [options[:cmd]]
+        if parallel?
+          parts = ['bundle exec parallel_rspec']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.